spec/features/create_gis_object_spec.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Create and accession GIS object', type: :feature do
+  let(:start_url) { "#{Settings.argo_url}/registration" }
+  let(:project_name) { 'GIS Integration Test' }
+  let(:source_id_random_word) { "#{random_noun}-#{random_alpha}" }
+  let(:source_id) { "gis-integration-test:#{source_id_random_word}" }
+  let(:label_random_words) { random_phrase }
+  let(:object_label) { "gis integration test #{label_random_words}" }
+  let(:collection_name) { 'Integration Test Collection - GIS' }
+  # this APO must exist on argo-stage and argo-qa and have "Integration Test Collection - GIS" collection available
+  let(:apo_name) { 'APO for GIS' }
+
+  before do
+    authenticate!(start_url: start_url,
+                  expected_text: 'Register DOR Items')
+  end
+
+  scenario do
+    # register new GIS object
+    select apo_name, from: 'Admin Policy'
+    select collection_name, from: 'Collection'
+    select 'file', from: 'Content Type'
+    fill_in 'Project Name', with: project_name
+    fill_in 'Source ID', with: source_id
+    fill_in 'Label', with: object_label
+
+    click_button 'Register'
+
+    # wait for object to be registered
+    expect(page).to have_text 'Items successfully registered.'
+
+    bare_object_druid = find('table a').text
+    druid = "druid:#{bare_object_druid}"
+
+    # # puts druid # useful for debugging
+
+    # Go to kurma server and copy test content to the druid folder so it can be accessioned
+    test_data_source_folder = File.join(Settings.gis.robots_content_root, 'integration_test_data')
+    test_data_destination_folder = File.join(Settings.gis.robots_content_root, bare_object_druid, 'temp')
```

Comment:
note: this integration test data is currently just sitting on the server...another alternative is to place it in this codebase and then SCP up to the server each time it is run